ui/src/app/store/organization/effects.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2020 the original author or authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+import {Store} from '@ngrx/store';
+import {Injectable} from '@angular/core';
+import {Actions, Effect, ofType} from '@ngrx/effects';
+import { switchMap } from 'rxjs/operators';
+import {OrganizationEnService} from '../organization/organization-entitys.service';
+import {AppState} from '../index';
+import { loadOrganizations } from './action';
+
+@Injectable()
+export class OrganizationEffect {
+    constructor(
+        private actions: Actions,
+        private organizationEnService: OrganizationEnService,
+        private store: Store<AppState>
+    ) {
+    }
+
+    @Effect({ dispatch: false })
+    fetchOrganizations =
+        this.actions.pipe(
+            ofType(loadOrganizations),
+            switchMap((action) => this.organizationEnService.getAll())
```

Comment:
please add error handling, and effect listening for error and showing snackbar message in case of error